packages/smart-wallet/src/walletFactory.js

```diff
@@ -6,10 +6,19 @@
  */
 
 import { BridgeId } from '@agoric/internal';
+import { fit, M } from '@agoric/store';
 import { makeAtomicProvider } from '@agoric/store/src/stores/store-utils.js';
 import { makeScalarBigMapStore } from '@agoric/vat-data';
 import { E, Far } from '@endo/far';
 import { makeSmartWallet } from './smartWallet.js';
+import { shape } from './typeGuards.js';
+
+const PrivateArgsShape = harden(
+  M.split(
+    { storageNode: M.eref(M.any()) },
+    M.partial({ bridgeManager: M.eref(M.any()) }),
```

Comment:
Should be `M.opt`, except it may also allow the property to be present but `undefined`. As is, this allows any additional properties in the shape.